.github/workflows/validate-nginx-config.yml

```diff
@@ -0,0 +1,74 @@
+name: Validate Nginx Configuration
+on:
+  workflow_call:
+    inputs:
+      custom_template_values:
```

Comment:
because now we pass all the required values to this input, I think `template_values` is sufficient.